src/pages/PersonaCreation/Page/Page.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import _ from 'lodash';
+import { Editor as EditorType } from 'slate/dist';
+
+// Import the Slate editor factory.
+import { createEditor } from 'slate';
+
+// Import the Slate components and React plugin.
+import { Slate, Editable, withReact, ReactEditor } from 'slate-react';
+import { PageWrapper } from 'components/PageWrapper';
+import styled from 'styled-components';
+import { TopNav } from 'components/TopNav/TopNav';
+import { BlockTools } from './BlockTools/BlockTools';
+import { Element } from './BlockTools/EditorFunctionalities/EditorFunctionalities';
+import { EditorContextProvider, EditorContextType } from './EditorContext';
+
+const StyledPageWrapper = styled(PageWrapper)`
+  position: relative;
+  padding: 0px;
+`;
+
+type ActiveToolsType = 'bloc' | 'inline' | false;
+
+export const Page = () => {
+  const [activeTools, setActiveTools] = useState<ActiveToolsType>(false);
+  const editor: EditorType = useMemo(() => withReact(createEditor()), []);
+  const renderElement = useCallback(props => <Element {...props} />, []);
```

Comment:
why deps array is empty?

Reply:
Because we operate on reference here and it should not be updated. According to official slate documentation.